lib/api/service/authentication_service.dart

```diff
@@ -16,5 +18,9 @@ abstract class AuthenticationServiceImpl extends AuthenticationService {
 
   @override
   @POST('/oauth/token')
-  Future<LoginResponse> logIn(@Body() LoginRequest request);
+  Future<AuthResponse> logIn(@Body() LoginRequest request);
+
+  @override
+  @POST('/oauth/token')
+  Future<AuthResponse> getAuthToken(@Body() RefreshTokenRequest request);
```

Comment:
Would this be easier to understand?
```suggestion
  Future<AuthResponse> refreshToken(@Body() RefreshTokenRequest request);
```